specification/attributes/datetime_requirements.md

```diff
@@ -0,0 +1,34 @@
+# Datetime Requirements
+
+Billing data columns that provide date and time information (datetime related columns) conforming to specified rules and formatting requirements ensure clarity, accuracy and ease of interpretation for both humans and systems.
```

Comment:
nit: We shouldn't need to say "billing data" since all date/time columns would follow this, right? Also, the second datetime reference seems redundant.
```suggestion
Columns that provide date and time information conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
```

Reply:
suggested changes applied